pkg/timeutils/utils_test.go

```diff
@@ -36,9 +52,37 @@ func TestGetTimestamp(t *testing.T) {
 	}
 
 	for _, c := range cases {
-		o := GetTimestamp(c.in, now)
+		o, err := GetTimestamp(c.in, now)
 		if o != c.expected {
-			t.Fatalf("wrong value for '%s'. expected:'%s' got:'%s'", c.in, c.expected, o)
+			if err != nil {
+				t.Fatalf("wrong value for '%s'. expected:'%s' got:'%s' with error: `%s`", c.in, c.expected, o, err)
+			}
+			t.Fatalf("wrong value for '%s'. expected:'%s' got:'%s' ", c.in, c.expected, o)
+		}
+	}
+}
+
+func TestParseTimestamps(t *testing.T) {
+	cases := []struct {
+		in                        string
+		def, expectedS, expectedN int64
+	}{
+		// unix timestamps
+		{"1136073600", 0, 1136073600, 0},
+		{"1136073600.000000001", 0, 1136073600, 1},
+		{"1136073600.0000000010", 0, 1136073600, 1},
+		{"1136073600.00000001", 0, 1136073600, 10},
+		{"foo.bar", 0, 0, 0},
```

Comment:
Not sure I understand this test but, shouldn't `ParseTimestamps` return `err` in this case ?